src/common/components/entry-list-item/index.tsx

```diff
@@ -189,7 +189,7 @@ export default class EntryListItem extends Component<Props, State> {
         const cls = `entry-list-item ${promoted ? "promoted-item" : ""}`;
 
         return (
-            <div className={_c(cls)}>
+            <div className={_c(cls)} id={entry.permlink.replace(/[0-9]/g, '')}>
```

Comment:
for unique id, always use `${entry.author}-${entry.permlink}` author and permlink together.